src/routes/Registration.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import Button from '../ui-kit/components/Button';
+
+const Container = styled.div`
+  margin: 16px;
+`;
+
+const Description = styled.div`
```

Comment:
Could we move those 3 style declarations to `/routes/styles.js` (separate file just for styles)

Reply:
For sure we can. But these are temporary styles and maybe it will be better to do this when the design will be comleted?

Reply:
ok, we will see later